### PR TITLE
[minor] adds radii-button token

### PR DESCRIPTION
**Pull request by agustisera1**
**Checklist**

**What is the ticket / issue associated with this change?** 
- https://impulsumvc.atlassian.net/browse/COM-1134

**Screenshot**
image

### CODE REVIEW

**Comment by guilean** [comment]
@agustisera1 Don't we have to add the token to `packages/components/src/theme/ComposerTheme/Components/Button.ts`?

File: packages/components/src/theme/ComposerTheme/Radius/radius.ts [hunk 1]
```diff
@@ -10,4 +10,5 @@ export const defaultRadius = {
   'radii-modal': '0.5rem',
   'radii-alertdialog': '0.5rem',
   'radii-popover': '0.5rem',
+  'radii-button': '0.375rem',
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Radius/radius.ts:13]: you have to use this value in the button theming

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/components/src/theme/ComposerTheme/Radius/radius.ts (hunk 1 above)
Comment (agustisera1) [reply packages/components/src/theme/ComposerTheme/Radius/radius.ts:13]: @tkanzakic Oops, forgot that
cc: @guilean

File: packages/components/src/theme/ComposerTheme/Components/Button.ts [hunk 2]
```diff
@@ -95,9 +95,9 @@ const generateAltButton = (color: string, theme: Record<string, any>) => {
 const $spinnerSize = cssVar('spinner-size');
 
 export const Button: ComponentStyleConfig = {
-  baseStyle: {
-    borderRadius: '0.375rem',
-  },
+  baseStyle: ({ theme }) => ({
+    borderRadius: theme.radii['radii-button'] || '0.375rem',
```
Comment (guilean) [inline packages/components/src/theme/ComposerTheme/Components/Button.ts:99]: Should we remove the default value?

Comment (agustisera1) [reply packages/components/src/theme/ComposerTheme/Components/Button.ts:99]: @guilean An outdated `theme.json` from contentful, could break the radius?

File: packages/components/src/theme/ComposerTheme/Components/Button.ts [hunk 3]
```diff
@@ -95,9 +95,9 @@ const generateAltButton = (color: string, theme: Record<string, any>) => {
...
+  }),
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/Button.ts:100]: I think you only need to do this:

```suggestion
  baseStyle: {
    borderRadius: 'radii-button',
  },
```

**Review by tkanzakic — COMMENTED** [review]
there are other borderRadius in the theme that should be updated too

File: packages/components/src/theme/ComposerTheme/Components/Button.ts [hunk 4]
```diff
@@ -96,7 +96,7 @@ const $spinnerSize = cssVar('spinner-size');
 
 export const Button: ComponentStyleConfig = {
   baseStyle: {
-    borderRadius: '0.375rem',
+    borderRadius: 'radii-button',
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/Button.ts:99]: we need to update line 90 too

File: packages/components/src/theme/ComposerTheme/Components/Button.ts [hunk 5]
```diff
@@ -87,7 +87,7 @@ const generateAltButton = (color: string, theme: Record<string, any>) => {
       bottom: '-1px',
       left: '-1px',
       border: `1px solid var(--chakra-colors-text-link-${color}-default)`,
-      borderRadius: theme?.components?.Button?.baseStyle?.borderRadius || '0.375rem',
+      borderRadius: theme?.components?.Button?.baseStyle?.borderRadius || 'radii-button',
```
Comment (tkanzakic) [inline packages/components/src/theme/ComposerTheme/Components/Button.ts:90]: just use the token

**Review by tkanzakic — APPROVED** [review]

**Review by guilean — APPROVED** [review]

**Pull request merged**